CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Release Notes
 
+## Version 1.32.0 - 2023-11-07
+
+Features:
```

Comment:
lets add the Beta changs into the release notes for the 32 stable (but do not add it it to the changelog)
so only in release notes we have combined version of beta and stable release notes. so the users get the news about new features

Reply:
Will do